Compfiles/Usa1979P1.lean

```diff
@@ -0,0 +1,50 @@
+/-
+Copyright (c) 2024 The Compfiles Contributors. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Hongyu Ouyang
+-/
+
+import Mathlib.Tactic
+
+import ProblemExtraction
+
+problem_file { tags := [.Algebra, .Inequality] }
+
+/-!
+# USA Mathematical Olympiad 1979, Problem 1
+
+Determine all non-negative integral solutions (n₁, n₂, ..., n₁₄) if any,
+apart from permutations, of the Diophantine Equation $n₁⁴ + n₂⁴ + ... + n₁₄⁴ = 1599.
```

Comment:
Unclosed `$`. Please either delete it, or do everything in LaTeX style.

Reply:
Changed to LaTeX style.